openstef_dbc/services/write.py

```diff
@@ -113,14 +114,16 @@ def write_forecast(
 
         # If desired, write tAhead series to influx - tAhead table
         if t_ahead_series:
+            if len(forecast) == 0:
+                self.logger.info("Len forecasts=0, not going to write them to tAheads")
+                return message
```

Comment:
Should this info message be included to 'message' before returning?

Reply:
Yes, well spotted, I changed this now